draft-ietf-quic-recovery.md

```diff
@@ -1030,14 +1030,17 @@ application limited or after a retransmission timeout. In order to limit
 the size of bursts sent into the network, the behavior when restarting from
 idle depends upon whether pacing is used.
 
-If the sender uses pacing, the connection should limit the initial burst of
-packets to no more than the initial congestion window and subsequent packets
-SHOULD be paced. The congestion window does not change while the connection
-is idle.
+If the sender uses pacing, the congestion window MUST not increase and does not
```

Comment:
Possibly the "MUST not increase CWND" part should move a paragraph up, since it's really not pacing specific?